[MTM-46145]-DOC-Add-requirements-for-managing-business-rules (#1042)
SoftwareAG/c8y-docs: content/users-guide/administration-bundle/business-rules.md

Alarm mapping enables you to change the severity and text of alarms to adapt them to your business priorities. For example, a loss of the connection to a device is by default a MAJOR alarm but may be critical to you. To change this, add an alarm mapping to change alarms related to connection losses to CRITICAL.

{{< c8y-admon-req >}}
ROLES & PERMISSIONS:

> - To view alarm mappings: READ permission for the permission type "Alarms".

Author: It's changed already.

Alarm mapping enables you to change the severity and text of alarms to adapt them to your business priorities. For example, a loss of the connection to a device is by default a MAJOR alarm but may be critical to you. To change this, add an alarm mapping to change alarms related to connection losses to CRITICAL.

> {{< c8y-admon-req >}}

Author: What standard roles do you mean?

This "part" is about `ROLES & PERMISSIONS`, the most important things which are needed to get access for alarm mapping. I think that `ROLES & PERMISSIONS` should be clear enough and we should avoid giving too many information.

> {{< c8y-admon-req >}}

Reviewer: I think @Airdroplet meant to list the default roles which contain the listed permissions. That way users will know which of the default roles they can use, instead of needing to go to each one and checking its details. See other PRs for how it looks like: https://github.com/SoftwareAG/c8y-docs/pull/1043/files#diff-8511b0768c9442e6f93b06633840e8caa6e40bd3617b32e066e7c3c030696251R24-R25

> {{< c8y-admon-req >}}

Author: Thanks for a hint! I'm going to check it!

ROLES & PERMISSIONS:

> To view alarm mappings: READ permission for the permission type "Alarms".
> 
> To manage alarm mappings (add, edit, or delete): ADMIN permission for the permission type "Alarms".

Reviewer 2: ```suggestion

- To view alarm mappings: READ permission for the permission type "Alarms".
- To manage alarm mappings (add, edit, acknowledge, clear or delete): ADMIN permission for the permission type "Alarms".
```

Reviewer 2: Please verify, if for acknowledge and clear ADMIN permission required, I'm not sure.

> To view alarm mappings: READ permission for the permission type "Alarms".
> 
> To manage alarm mappings (add, edit, or delete): ADMIN permission for the permission type "Alarms".

Reviewer 2: ```suggestion
- To view alarm mappings: READ permission for the permission type "Alarms".
- To manage (add, edit, or delete) alarm mappings: ADMIN permission for the permission type "Alarms".
```

Alarm mapping enables you to change the severity and text of alarms to adapt them to your business priorities. For example, a loss of the connection to a device is by default a MAJOR alarm but may be critical to you. To change this, add an alarm mapping to change alarms related to connection losses to CRITICAL.

{{< c8y-admon-req >}}
ROLES & PERMISSIONS:

- To view alarm mappings: READ permission for the permission type "Option management".
- To manage (create, edit, or delete) alarm mappings: ADMIN permission for the permission type "Option management".

For easier user access management, the above permissions are included in the global role created by default in every new tenant:

> - Tenant manager - manages tenant wide configurations like applications, tenant options and Business rules.

Reviewer 3: ```suggestion
- Tenant manager - manages tenant-wide configurations like applications, tenant options and business rules.
```

> - Tenant manager - manages tenant-wide configurations like applications, tenant options and business rules.

Reviewer: Keep "Tenant Manager" as agreed in https://github.com/SoftwareAG/c8y-docs/pull/1035#discussion_r1198979392.

Author: Changed again.
Done.

> The above permissions can be used to display Alarm mapping in Business rules. Every new tenant have specified typical roles by default:
> - Tenant Manager - Can manage tenant wide configurations like applications, tenant options and Business rules.

Reviewer: @BeateRixen, @Airdroplet , do we apply formatting for "Alarm mapping" or "Business rules"?
Not sure about grammar here: "Every new tenant has typical roles specified by default:" (but we list only one role, so rather "typical role"). Or this introductory sentences - they should explain the link between permission(s) listed above and global role(s) listed below. Maybe:
```suggestion
For easier access management, the above permissions are included in the global role created by default in every new tenant:
- Tenant Manager - Can manage tenant wide configurations like applications, tenant options and Business rules.
```

Reviewer 3: alarm mapping and business rules are both not capitalized and not formatted.

Author: Ok.

> - Tenant Manager - Can manage tenant wide configurations like applications, tenant options and Business rules.

Reviewer 3: ```suggestion
- Tenant Manager - Can manage tenant-wide configurations like applications, tenant options and business rules.
```

Reviewer: @pmich-sag, I've unresolved this thread as @BeateRixen's suggestion is not applied.

Author: Done.
